metrics_utility/automation_controller_billing/package/package_crc.py

```diff
@@ -38,7 +39,15 @@ def _get_rh_password(self):
         return os.getenv('METRICS_UTILITY_SERVICE_ACCOUNT_SECRET')
 
     def _get_http_request_headers(self):
-        return get_awx_http_client_headers()
```

Comment:
this change is also unrelated, and would cause us to always report `UNLICENSED`... but .. you do have a perfectly working version of `get_license_info_from_db` you could use here, right?

Also, `version('metrics-utility')` is not the same thing as `get_awx_version` ... and you have `get_controller_version_from_db` which does have the right info

Reply:
ok I'm going to remove this file from this pr for now.